fastlane_core/lib/fastlane_core/cert_checker.rb

```diff
@@ -124,8 +123,8 @@ def self.install_wwdr_certificate(url)
 
     def self.wwdr_keychain
       priority = [
-        "security list-keychains -d user",
-        "security default-keychain -d user"
+        "security default-keychain -d user",
+        "security list-keychains -d user"
```

Comment:
I'm not sure about this change, according to https://github.com/fastlane-old/fastlane_core/pull/116 we should start with the login keychain - if we want to move default up, maybe we should insert `security login-keychain` at the very top?

Reply:
Login keychain is default, most of the time. Also if you have keychain named like aaaa.keychain-db then login one will be not picked here at all. 
From me it has more sense to start default one, because in most cases its used by match and other tools.

Reply:
@getaaron 
This old setup is based on this comment
https://github.com/fastlane-old/fastlane_core/pull/116#issuecomment-190934610
Which is not true. Default-keychain is one set by user from gui or `security default-keychain -s <keychain_name>`.
There are chances that it's not set at all, but then we will pick any from list-keychains.